sktime/alignment/dtw_python.py

```diff
@@ -0,0 +1,337 @@
+# -*- coding: utf-8 -*-
+"""Interface module to dtw-python package.
+
+Exposes basic interface, excluding multivariate case.
+"""
+
+__author__ = ["fkiraly"]
+
+import pandas as pd
+import numpy as np
+
+from sktime.utils.validation._dependencies import _check_soft_dependencies
+from sktime.alignment._base import BaseAligner
+
+_check_soft_dependencies("dtw")
+
+
+class AlignerDTW(BaseAligner):
+    """Aligner interface for dtw-python.
+
+    Behaviour: computes the full alignment between X[0] and X[1]
+        assumes pairwise alignment (only two series) and univariate
+        if multivariate sequences are passed:
+            alignment is computed on univariate series with variable_to_align;
+            if this is not set, defaults to the first variable of X is used
+        raises an error if variable_to_align is not present in X[0] or X[1]
+
+    Parameters
+    ----------
+    dist_method: string, pointwise (local) distance function to use
+        one of the functions in `scipy.spatial.distance.cdist`
+        default = 'euclidean'
```

Comment:
* I dont think the phrase `pointwise (local) distance function` is clear / unambiguous.
* What does word local mean? From pure maths, local tends to mean that there exists a neighbourhood in which things are defined/valid. I dont think thats what is going on here.
* Giving the signature of distance function would help, e.g. `pointwise distance function with signature (float, float) -> float`